apps/sparrow/settings/local.py

```diff
@@ -35,5 +36,12 @@ def show_toolbar(request):
     'SHOW_TOOLBAR_CALLBACK': "sparrow.settings.local.show_toolbar",
 }
 
+JENKINS_TASKS = (
+    'django_jenkins.tasks.run_pep8',
+    'django_jenkins.tasks.run_pyflakes',
+    'django_jenkins.tasks.run_jshint',
+    'django_jenkins.tasks.run_csslint'
+)
```

Comment:
Right. I somehow thought it was separate action, probably because I was thinking about SLOCCount.